backend/store/etcd/check_store.go

```diff
@@ -26,6 +25,17 @@ func GetCheckConfigsPath(ctx context.Context, name string) string {
 	return checkKeyBuilder.WithContext(ctx).Build(name)
 }
 
+func schedulerFor(c *corev2.CheckConfig) string {
+	if c.Scheduler == "" {
+		if c.RoundRobin {
+			return "etcd"
```

Comment:
Should we use the constants here? e.g. `corev2.EtcdScheduler`

Reply:
Yup, absolutely.